vuepress/docs/next/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -50,36 +51,45 @@ mkdir testProject && cd testProject
 ent jhipster --blueprints entando
 ```
 
-3. You'll be presented with a series of prompts to configure your project. These are listed below, with the base values for this tutorial in parentheses. Except where noted in **bold**, you can input what works best for you. Note that hitting "Enter" will select the default option.
+3. This triggers a project initialization prompt, which you should affirm:
```

Comment:
this is in addition and prior to the series of prompts. it was omitted from the original page. this prompt is wrt project initialization, which can only be one thing and is explicitly listed

Reply:
changed "affirm" ;)

Reply:
i see, did not see the difference.

Reply:
yeah it was annoying and concerning when my env didn't exactly mimic the tutorial so i added this; idk if it's strictly necessary because it should be obv you need to initialize, but i didn't want to start some kind of subjective cherry-picking of what needs to be mentioned so threw it in with the rest of the kitchen sink